### PR TITLE
#5375 user:cancel to support user_cancel_reassign method by default

### PR DESCRIPTION
**Pull request by stratulat-vladislav**

### CODE REVIEW

**Comment by weitzman** [comment]
I think this command description should be "Block user account and optionally delete or reassign its content". Lets add a new boolean option for reassign. Its description should say that content will belong to the Anonymous user.

**Comment by stratulat-vladislav** [comment]
Why do we have to mention "Block" on **cancel** command while we have user:block command for this task?

**Comment by stratulat-vladislav** [comment]
The documentation says: **Cancel the user account with the name username and anonymize all content created by that user.**, but it doesn't do that.
https://www.drush.org/latest/commands/user_cancel/

**Comment by weitzman** [comment]
Because thats how it has historically worked and I'm not inclined to surprise folks with a behavior change.

**Comment by stratulat-vladislav** [comment]
> I think this command description should be "Block user account and optionally delete or reassign its content". Lets add a new boolean option for reassign. Its description should say that content will belong to the Anonymous user.

Hi @weitzman. I have pushed further changes. Can you please have a look?

File: src/Drupal/Commands/core/UserCommands.php [hunk 1]
```diff
@@ -121,7 +121,11 @@ public function renderRolesCell($key, $cellData, FormatterOptions $options)
      * @param string $names A comma delimited list of user names.
      * @aliases ublk,user-block
      * @usage drush user:block user3
-     *   Block the users whose name is user3
+     *   Block the user whose name is <info>user3</info>
```
Comment (stratulat-vladislav) [inline src/Drupal/Commands/core/UserCommands.php:124]: Hi @weitzman. I have updated help text and usage examples as you did for 11.x branch.

**Comment by weitzman** [comment]
Unfortunately, the 10.x branch is EOL as well. Tests have broken it seems so it wouldnt be prudent to merge this. Thanks for working on older versions of Drush.

**Pull request closed without merging**